backend/vxlan/vxlan.go

```diff
@@ -19,25 +20,28 @@ const (
 	vxlanInterfaceName = "vtep1042"
 	vxlanVni           = 1042
 	vxlanMACRange      = "0E:00:00:00:00:00"
-	vxlanMTU           = 1400
+	vxlanMTU           = 1500
```

Comment:
Can you please make this configurable? Take it from the catalog question? Are you able to set it to values greater than 1500? This is important on 10G networks.